### DISCUSSION TITLE
Embedding training on internal LSM platform

### DISCUSSION THREAD

**Discussion by kingabaran** [Green-Software-Foundation/learn#178] [Q&A]
Hi, I am working for WithSecure company, cyber security solutions vendor. 
Is it possible to receive course material in SCORM format. 2004 (any version of that)?
We would love to spread green software practices with new joiners of our organization and for this purpose we have internal LMS   portal where we can track completion of courses and guide members of organization properly.

We would be interested in exam as well in whatever text format you can provide.
Having it built-in on our platform we have also proper reporting possibilities.

Please let me know if this approach can work?
Kinga

**Comment by russelltrow** [comment]
Hello @kingabaran,

Apologises for such a delayed response on this point, I have recently joined the GSF as a PM and picked up our training course.

We are working on a roadmap for the course and a SCORM version is definitely on there. We are still finalising exactly how and when this is likely to be available. Once I have further details I will come back to you and we can discuss how to proceed.

Russ

**Comment by kingabaran** [comment]
Hi Russel,
Thank you and appreciate letting me know when situation changes.

BR
Kinga

**Comment by kingabaran** [comment]
Hi Russel,

Coming one more time to the topic, do you have maybe more clarity when to expect SCORM version of this course?
In case you would respond no..;) I wanted to ask also about some reporting out of your system as far as time spent on training by users from WithSecure are concerned.

I noticed below information on the page – is that the way to possess reporting I mentioned?
Some employers have created forms to facilitate the administration of credits for completing this course. If your employer is in this list<https://docs.google.com/document/d/1Jv-f7wGyODzEySQFfVk6Vo_OAHWjf-jAdFmyfDQUgvU/edit?usp=sharing>, please click on your organization name to register your credits. If your organization would like to add their form to the list, please get in touch with our team at ***@***.******@***.***>.

I wish you a good day
Kinga


From: Kinga Baran
Sent: Tuesday, September 12, 2023 1:21 PM
To: Green-Software-Foundation/learn ***@***.***>; Green-Software-Foundation/learn ***@***.***>
Cc: Mention ***@***.***>
Subject: RE: [Green-Software-Foundation/learn] Embedding training on internal LSM platform (Discussion #178)

Hi Russel,
Thank you and appreciate letting me know when situation changes.

BR
Kinga

**Comment by russelltrow** [comment]
Hello Kinga, 

We are still very much planning to produce a SCORM package for the course but I'm sorry to say I still don't have a better timeline than early 2024.

I can absolutely help you with reporting, could you please drop an email to russell[at]greensoftware.foundation and we can coordinate.

Russ
